code/game/objects/items/food/egg.dm

```diff
@@ -85,6 +85,30 @@
 	else
 		..()
 
+/obj/item/food/egg/afterattack_secondary(atom/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
+	if(. == SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN)
+		return
+
+	if(!istype(target, /obj/machinery/griddle))
+		return SECONDARY_ATTACK_CALL_NORMAL
+
+	var/atom/broken_egg = new /obj/item/food/rawegg(target.loc)
+	broken_egg.pixel_x = pixel_x
+	broken_egg.pixel_y = pixel_y
+	playsound(get_turf(user), 'sound/items/sheath.ogg', 40, TRUE)
+	reagents.copy_to(broken_egg,reagents.total_volume)
+
+	var/obj/machinery/griddle/hit_griddle = target
+	hit_griddle.AddToGrill(broken_egg, user)
+	target.balloon_alert(user, "You crack [src] open onto [target].")
```

Comment:
```suggestion
	target.balloon_alert(user, "cracked open egg")
```
Balloon alerts are written as concise, less-specific, and punctuated (except for failure) messages that generally don't need to spam up your chat

Reply:
why not cracked open [src]?

Reply:
you never know when someone adds "willy nimble french bird quail egg (imported) (real)" egg subtypes and suddenly your balloon alert is mega long